extensions/stackdriver/config/v1alpha1/stackdriver_plugin_config.proto

```diff
@@ -18,19 +18,17 @@ syntax = "proto3";
 package stackdriver.config.v1alpha1;
 
 message PluginConfig {
-  // Indicates whether the module should act as inbound or outbound reporter.
-  // Difference being:
-  //   * inbound and outbound have different metric types to export.
-  //   * inbound reporter writes access logs.
-  enum ReporterKind {
-    INBOUND = 0;
-    OUTBOUND = 1;
-  }
-  ReporterKind kind = 1;
-
   // Controls whether to export access log.
-  bool disable_access_logging = 2;
+  bool disable_access_logging = 1;
```

Comment:
nit: should we strive to have positive booleans (ex: `enable_access_logging` )?

Reply:
proto3 does not allow default value, and we want to enable access log by default with zero config, so `disable_access_logging` might be a better fit here?

Reply:
we can add an empty embedded message as a control. the message can be populated with other access log settings down the line.

Reply:
Are you suggesting something like
```
message PluginConfig {
  message AccessLogControl {}
  AccessLogControl access_log_control = 1;
}
```
Then if you want to turn on access log, you still need to specify an empty struct in the config right? "accessLogControl: {}". It does look better to have this for any future access log related settings, though I am not sure what will they be...

Reply:
Yeah. The value depends whether we expect this accesslogcontrol config to grow. If not, it doesn't really matter.